### DISCUSSION TITLE
Notes  / Comments

### DISCUSSION THREAD

**Discussion by vmitchell85** [spatie/flareapp.io-roadmap#48] [Ideas]
I would love to be able to add some notes/comments to an Exception entry. Leaving notes about when it will be fixed, or links to our issue tracker, etc.

Thanks!
- Vince

**Comment by kaiserkiwi** [comment]
This would be awesome. We always make tickets out of the errors Flare shows us but it's extremely difficult to keep track of this because we can't put the ticket link or ticket number on any error.

**Comment by freekmurze** [comment]
We only retain errors in Flare for a certain period, so if we would directly add notes, those notes could get deleted as well.
In the Flare UI you create/connect GitHub issues (we added that a couple of months ago). We're using that ourselves whenever we want to comment on an error

Is that a strategy that could work for you as well @vmitchell85 @kaiserkiwi ?

**Reply by vmitchell85** [reply]
Having a note that gets deleted with the error I think would work better than nothing.

I didn't really consider retention when submitting this discussion. We currently use ClickUp to manage our issues. It would be great if there was some way to export an error with the full trace and data so we could reference that.

There are some cases where we can't get an error resolved in 30 days so then our link to the Flare error wouldn't work anymore.

Side Note: I know Flare isn't open-source, but if there is any way I can help with a ClickUp integration I'd be willing to contribute.

**Reply by kaiserkiwi** [reply]
> We only retain errors in Flare for a certain period, so if we would directly add notes, those notes could get deleted as well.

This would be perfectly fine. As our experience shows the errors get only deleted if they aren't occurred again the last 30 days. If we have this situation the error is probably fixed. No need for historical data.

**Comment by jerodev** [comment]
I had a related idea to add a comment when resolving, snoozing, or closing an issue. Not sure if I should start a new discussion for this?

This could also be propagated to GitHub issues. This way you can know why or how an issue is resolved or closed.

**Comment by jtaylor69** [comment]
We've started using Flare and had the same suggestion raised by our developers. We don't care that the notes don't persist, but it'd be nice to leave note in the way of pointers to say that "It's being looked at", or "It's because of X" before it makes it way through to our project management tools. We don't use GitHub and use Bitbucket so can't make use of the automatic issues, but we can probably utilise webhooks to do similar if we wanted to.

**Comment by rubenvanassche** [comment]
https://flareapp.io/blog/61-unveiling-error-notes-and-the-activity-feed

It has arrived, let us know what you think!
